runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -1666,13 +1666,78 @@ TR_VectorAPIExpansion::transformIL(bool checkBoxing)
          continue;
 
       bool scalarized;
-      bool vectorizedNode;
-
+      bool vectorizedOrScalarizedNode;
```

Comment:
You can declare and initialize the variable in a single line by moving this declaration to L.1674.